tests/sentry/tasks/test_weekly_reports.py

```diff
@@ -634,7 +634,7 @@ def test_message_builder_multiple_users_prevent_resend(self, message_builder, re
 
         # TODO(RyanSkonnord): Make sure this doesn't cause false negatives after
         #  batch IDs are also used to prevent duplicate sends
-        batch_id = UUID("ea18c80c-d44f-48a4-8973-b0daa3169c44")
+        batch_id = str(UUID("ea18c80c-d44f-48a4-8973-b0daa3169c44"))
```

Comment:
```suggestion
        batch_id = "ea18c80c-d44f-48a4-8973-b0daa3169c44"
```
seems like we could skip some casting.